lessonThree.py

```diff
@@ -10,50 +10,67 @@
 # If the password for that user doesn't match then it should return "password is incorrect"
 
 
-users = []
+users = ["Alayna", "Lance", "Grayson", "Vivian"]
 
 def createUser(fullName, email, userName, password, phoneNumber) :
     # your code here
+    user = []
+    
+    for user in user :
+      user.append({
+        'user': 'user', 
```

Comment:
This is just appending the word "user" to the property `user`. You need to append all the parameters in the function call to their respective property names.